### DISCUSSION TITLE
Support for older Java versions

### DISCUSSION THREAD

**Discussion by Lonzak** [LibrePDF/OpenPDF#1052] [General]
In the issue #1049 it was mentioned that there is the possibility to support older java versions:

> It seems that some users are not able to update their Java versions so easily, and for these users we can maintain separate branches for old Java version, if you specifically request it. Over time most users are able to update their Java versions eventually.

That would be really appreciated and the only thing I wanted to suggest with my posts (#1033 #1034 ...). So if we could maintain a java8 branch that would really help. I think we can delete the java7 branch - that one no one needs anymore...

**Comment by andreasrosdalw** [comment]
Contact asturio if you want to help maintain support for older Java versions.
